app/models/user.rb

```diff
@@ -0,0 +1,20 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+
+  has_many :photos
+  has_many :likes
+
+
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable, :omniauthable, :omniauth_providers => [:facebook]
```

Comment:
Metrics/LineLength: Line is too long. [113/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)